lib/dependencies.sh

```diff
@@ -29,6 +29,36 @@ run_if_present() {
   fi
 }
 
+log_build_scripts() {
+  local build=$(read_json "$BUILD_DIR/package.json" ".scripts[\"build\"]")
+  local heroku_prebuild=$(read_json "$BUILD_DIR/package.json" ".scripts[\"heroku-prebuild\"]")
+  local heroku_postbuild=$(read_json "$BUILD_DIR/package.json" ".scripts[\"heroku-postbuild\"]")
+
+  if [ -n "$build" ]; then
+    mcount "scripts.build"
+
+    if [ -z "$heroku_postbuild" ]; then
+      mcount "scripts.build-without-heroku-postbuild"
+    fi
+  fi
+
+  if [ -n "$heroku_prebuild" ]; then
+    mcount "scripts.heroku-prebuild"
+  fi
+
+  if [ -n "$heroku_postbuild" ]; then
+    mcount "scripts.heroku-postbuild"
+  fi
+
+  if [ -n "$heroku_postbuild" ] && [ -n "$build" ]; then
+    mcount "scripts.build-and-heroku-postbuild"
```

Comment:
I'd prefer to just keep this explicit for now. If I make the change to support `build` I'll revisit these.